app/models/answer.rb

```diff
@@ -0,0 +1,23 @@
+class Answer < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :question
+  has_many :comments, as: :response, dependent: :destroy
+  has_many :votes, as: :tally, dependent: :destroy
+
+  validates :content, presence: true
+  validates :user, presence: true
+  validates :question, presence: true
+  validate :content_must_not_be_nil
+
+
+  def vote_count
+    self.votes.inject(0) {|x, y| x + y.value}
+  end
+
+  def content_must_not_be_nil
+    if self.content.length == 0
```

Comment:
It's usually better to just check `unless self.content`. The reason is that if the value is `nil`, you still want to add the error, but `nil.length` will cause an exception.